lib/_http_server.js

```diff
@@ -90,6 +90,7 @@ let debug = require('internal/util/debuglog').debuglog('http', (fn) => {
 });
 
 const dc = require('diagnostics_channel');
+const storageChannel = new dc.StorageChannel('http.server');
```

Comment:
Little question: is this just an example for the PR, or do you plan to add a few in core modules?

Reply:
Not just an example, but this is only currently intended to exist here. Should probably be added to http2 also, and will need to use the interface externally in some places like job processors.